### ISSUE TITLE
Archinstall crashing when trying to create LUKS Encrypted Btrfs installation

### ISSUE THREAD

**Issue by 0xDezzy** [archlinux/archinstall#2735]
### Which ISO version are you using?

2024.10.01

### The installation log

https://0x0.st/XIum.log

#### Description of the issue

I was installing using a VMWare workstation VM with my second physical disk passed through to it

Then it failed when trying to actually install arch while trying to partition the disks.

**Comment by svartkanin** [comment]
Hmm I run an installation with your config and everything installed just fine, the encryption worked as expected. Looking at the config I also can't see anything that might cause the issue as your using the default layouts. 
Best answer I have atm is to run it again?

**Comment by 0xDezzy** [comment]
I took a deeper look and it was due to a residual LUKS header on one of the partitions. I overwrote the first gig of each partition to be sure and it worked after that, but then I encountered a different issue with the keyring sync. I noticed a separate issue about that but I'll focus on this issue here.

Is there a way we can make sure that the archinstall script formats the disk to prevent this issue when trying to install over an existing LUKS partition?
